spec/spec_helper_acceptance.rb

```diff
@@ -31,7 +17,6 @@
     # Install module and dependencies
     hosts.each do |host|
       copy_module_to(host, source: proj_root, module_name: 'nodejs')
-      shell("/bin/touch #{default['puppetpath']}/hiera.yaml")
       on host, puppet('module install puppetlabs-apt --version 2.0.1'), acceptable_exit_codes: [0, 1]
```

Comment:
should we remove the version foo here to get the latest?

Reply:
I'd consider https://github.com/puppetlabs/beaker-module_install_helper since that also takes are of the ```copy_module_to``` line above if you're going to fix that.

Reply:
Fine, I improved it and now you want more improvements? 😛 Ok, sold